exporter/awskinesisexporter/internal/batch/model.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto2";
```

Comment:
Not sure I follow why we need the proto definition?

Could you comment it further and make reference to any external resources as well?

Reply:
We don't need it but I saw it is a command practice to reference it. maybe I will add it in a readme file or add a link to https://github.com/awslabs/amazon-kinesis-producer/blob/master/aggregation-format.md instead.